File: FuncamentosProgramacao/CapituloTres/AtividadeQuinze.cs
namespace FuncamentosProgramacao.CapituloTres;

public class AtividadeQuinze
{
    public static void CalculaRestoSalario()
    {
        double salario;
        double contaUm;
        double contaDois;
        double restoSalario;

        Console.Write("Informe o salário da pessoa: ");
        salario = Double.Parse(Console.ReadLine()!);
        Console.Write("Informe o valor da primeira conta: ");
        contaUm = Double.Parse(Console.ReadLine()!);
        Console.Write("Informe o valor da segunda conta: ");
        contaDois = Double.Parse(Console.ReadLine()!);

        restoSalario = salario - ((contaUm * 1.02) + (contaDois * 1.02));

        Console.WriteLine($"O que sobra do salário da pessoa ao pagar as contas é: {restoSalario:F}");
    }
}

File: fundamentosProgramacaoJava/src/capitulo/tres/AtividadeQuinze.java
package capitulo.tres;

import java.util.Scanner;

public class AtividadeQuinze {

    public static void main(String[] args) {

        float salario;
        float restoSalario;
        float contaUm;
        float contaDois;
        Scanner leia = new Scanner(System.in);

        System.out.print("Digite o valor do salario de Joao: ");
        salario = leia.nextFloat();
        System.out.print("Digite o valor da prieira conta: ");
        contaUm = leia.nextFloat();
        System.out.print("Digite o valor da segunda conta: ");
        contaDois = leia.nextFloat();

        contaUm = (float) (contaUm * 1.02);
        contaDois = (float) (contaDois * 1.02);
        restoSalario = salario - (contaUm + contaDois);

        System.out.println("O salario restante de Joao e: R$" + restoSalario);

        leia.close();

    }

}
